README.md

```diff
@@ -53,6 +53,9 @@ If you find a problem or missing feature, open an issue or a pull requests on ht
 
 To have a higher chance of issue being solved, please attach a log file. To record one, go to `settings wheel -> System settings -> Logging -> Enable Debug Logging` and follow the procedure at http://kodi.wiki/view/Log_file/Easy
 
-## Note for Raspberry Pi users:
+# Note for ARM users #
+## Nvidia Shield / most Android boxes ##
+- _Ambilight mode_ doesn't properly work with some HD codecs (usually x265 >1080p) when "allow hardware acceleration - Mediacodec (Surface)" is enabled.
+## Raspberry Pi's: ##
  - Save the add-on configuration by exiting Kodi before shutting down the Pi completely
- - Ambilight mode doesn't work on a Raspberry Pi due to the way it renders video
+ - _Ambilight mode_ doesn't work on a _Raspberry Pi_ due to the way it renders video
```

Comment:
Thanks for all the info! :+1: Maybe add some of it in the `README.md`?

Also I can give some feedback on the Raspberry Pi 3:
I just followed the install instructions and it works totally fine for some videos. For some not.
I haven't done too much testing yet but it seems older (or non HD) codecs work and some newer HD videos might not...

If I find out more, I'll add it here later.
But you could change this line already to just say something like "it does not work fully on all Pis but that shouldn't stop you from trying, as it might work anyways and even if not, theater mode is already pretty cool" :wink: